### ISSUE TITLE
Accessibility: Editboxes in Signin Page not accessible. (Username/password)

### ISSUE THREAD

**Issue by ravarora** [AzureAD/azure-activedirectory-library-for-android#1211]
While signing in to work account using ADAL in Kaizala the user is redirected to a web page inside the app to enter User name and password. If Talkback is on double tapping into the username edit box does not brings up the software keyboard making it inaccessible to people using screen reader. 
<ul>
  <li>Expected behavior</li>
  <p>With Talkback on, on double tapping the "Enter Username" editbox, the software keyboard should come up. </p>

  <li>Actual behavior</li>
  <p>With Talkback on, on double tapping the "Enter Username" editbox, nothing happens. </p>
  <li>Steps to reproduce the problem</li>
  <ol>
     <li> Login and setup Kaizala. Make Sure Talkback is on</li>
     <li> Go to "Me" tab -> Linked Accounts -> Work Account</li>
     <li> keep swiping right in talkback enabled mode to reach "Get Started"</li>
     <li> User is redirected to Login Page. Swipe right to reach "enter username: edit box. </li>
     <li> Double ta[ when the focus is on "username" edit box</li>
     <li> The software keyboard doesnot pops up. </li>
  </ol></br>

  <li>Specifications like the version of the library, operating system etc</li>
  <ul>
      <li>ADAL Version : 1.13.3</li>
      <li>Android Version : 8.1.0</li>
      <li>Talk Back Version : 6.1.0.182429506.</li>
      <li>Application Version : 1.0.9404.1301(From Google play store)</li>
      <li>Device : Google Pixel XL</li>
  </ul></br>
</ul>

Please see the attached video for behavior. 

[ADAL Accessibility Issue.zip](https://github.com/AzureAD/azure-activedirectory-library-for-android/files/2052367/ADAL.Accessibility.Issue.zip)

**Comment by iambmelt** [comment]
Hi @ravarora -- I have confirmed the behavior you've reported in my own testing. This appears to be related the AAD Sign In controls presented in the WebView -- as such, I will engage the appropriate owner to address this accessibility issue. You will be CC'd once I have a contact for this issue.

**Comment by iambmelt** [comment]
In the meantime, there is a possible workaround for this issue which you can trigger by pressing `Next` without entering an email (See video [here](https://microsoft-my.sharepoint-df.com/:v:/p/brianmel/Ea4M3-_XWWJHoFWs8MOSSTcBJM1LAxsrrq6qu_sehCpLSg?e=0HLIxt))

**Comment by iambmelt** [comment]
Issue forwarded to AAD Portal UX team. Will reopen if native changes are req'd
